dev-docs/modules/freewheel.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page_v2
+page_type: module
+title: Module - Freewheel
+description: Returns targeting kev/value pairs for adpod mediaType adUnits.
+module_code : Freewheel
+display_name : Freewheel
+enable_download : true
+sidebarType : 1
+---
+
+# Freewheel Module
+
+{:.no_toc}
+
+This module passes key value targeting pairs to the Freewheel SDK. 
+
+## How to use the module:
+
+Send ad unit codes to the getTargeting method of this module. 
+
+#### Example:
+
+Ad Unit Code
+```
```

Comment:
Need a line break between the text and the code block here for the block to render.

Reply:
See my other comment vis-a-vis code blocks. Edited so hopefully it resolves.

Reply:
I made a change, let me know if it resolves for you.